README.md

```diff
@@ -57,7 +57,23 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker execution time) after the page has been closed by the user. Periodic Background Sync extends it to potentially infinite time thereafter. Here are some privacy concerns:
+
+* There is no notification to the user.
+* A periodic sync can be enabled while the user is connected to one network, and the sync event
+can be fired later when they're connected to another network. This can cause inadvertent leakage 
+of browsing history on an unintended network. This concern applies to other background tasks such as one-shot Sync and Background Fetch as well.
```

Comment:
"one-shot Sync" - you call this "one-shot Background Sync" before, it would be good to settle on something and be consistent about it.

Reply:
done.